packages/cubejs-snowflake-driver/driver/SnowflakeDriver.js

```diff
@@ -67,6 +69,17 @@ class SnowflakeDriver extends BaseDriver {
         WHERE columns.table_schema NOT IN ('INFORMATION_SCHEMA')
      `;
   }
+
+  async release() {
```

Comment:
As suggested for the parameter `clientSessionKeepAlive` here
https://docs.snowflake.com/en/user-guide/nodejs-driver-use.html#additional-connection-options